.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Dataflow_Java11.groovy

```diff
@@ -19,9 +19,11 @@
 import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 
+final String JAVA_11_HOME = '/usr/lib/jvm/java-11-openjdk-amd64'
+final String JAVA_8_HOME = '/usr/lib/jvm/java-8-openjdk-amd64'
 
 PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_Dataflow',
-  'Run Dataflow ValidatesRunner Java 11', 'Google Cloud Dataflow Runner ValidatesRunner Tests On Java 11', this) {
+    'Run Dataflow ValidatesRunner Java 11', 'Google Cloud Dataflow Runner ValidatesRunner Tests On Java 11', this) {
```

Comment:
Can we change job title to either:
a) trigger phrase
b) gradle target being executed.

Otherwise it doesn't give much additional information and loses a lot in functionality.

Reply:
I'm not sure if I understand you - we're using the PostcommitJobBuilder.postCommitJob() method here. It creates 2 jobs: 
- "beam_PostCommit_Java11_ValidatesRunner_Dataflow"
- "beam_PostCommit_Java11_ValidatesRunner_Dataflow_PR" (the suffix is added automatically by the builder - there's no way to modify it if you're using the builder)

What do you mean by "job title"? Do you mean the "name"? or the "triggerPhrase" or "githubUIHint" in the [postCommitJob](https://github.com/apache/beam/blob/master/.test-infra/jenkins/PostcommitJobBuilder.groovy#L47) method? In your opinion, should I separate the creation of the two jobs and have custom configurations (including naming) for the two?

Reply:
In general, I think we should stick with using the builders to enforce a common convention for all the jobs. Changing the convention and improving/refactoring the builders (in case we want to do this) should be part of different pr - this is already a huge one IMO and focuses on different aspects (of course a jira ticket should be created prior this).

Reply:
LMK if you have any comments or in case I misunderstood. :)

Reply:
I decided to remove the commit that introduced the change - I had some problems with Jenkins not reloading the job definition. An issue for the Jenkins jobs is created here: https://issues.apache.org/jira/browse/BEAM-8940